src/rpc/handlers/ServerDefinitions.h

```diff
@@ -0,0 +1,71 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#pragma once
+
+#include <etl/LoadBalancer.h>
+#include <rpc/common/Specs.h>
+#include <rpc/common/Types.h>
+
+namespace rpc {
+
+template <typename LoadBalancerType = etl::LoadBalancer, typename CountersType = rpc::Counters>
+class BaseServerDefinitionsHandler {
+    std::shared_ptr<LoadBalancerType> balancer_;
+
+public:
+    struct Output {
+        boost::json::object serverDefinitions;
+    };
+
+    using Result = HandlerReturnType<Output>;
+
+    explicit BaseServerDefinitionsHandler(std::shared_ptr<LoadBalancerType> const& balancer) : balancer_(balancer)
+    {
+    }
+
+    static RpcSpecConstRef
+    spec([[maybe_unused]] uint32_t apiVersion)
+    {
+        static const RpcSpec rpcSpec = {};
+        return rpcSpec;
+    }
+
+    Result
+    process(Context const& ctx) const
+    {
+        auto serverDefinitionsOpt =
+            balancer_->forwardToRippled({{"command", "server_definitions"}}, ctx.clientIp, ctx.yield);
```

Comment:
It seems there is a parameter `hash`. Probably we should accept any parameters and forward them.

Reply:
no need to forward the parameter manually . See the list of commands we shall forward here -> https://github.com/XRPLF/clio/blob/8ebe2d6a802e5a6c2b078b339fedcdfa08984524/src/rpc/common/impl/ForwardingProxy.h#L103
Add the new one here , the framework will do it for you ~